app/src/main/res/xml/preferences.xml

```diff
@@ -33,6 +33,14 @@
             android:summaryOn="@string/preference_enable_mobile_data_summary_on"
             android:title="@string/preference_enable_mobile_data_title" />
 
+        <ListPreference
+            android:defaultValue="640 X 480"
+            android:dialogTitle="Choose Resolution"
+            android:key="ImageUpload"
+            android:selectable="true"
+            android:summary="@string/prefs_image_upload"
+            android:title="Choose resolution" />
```

Comment:
Needs to be translatable, and something along the lines of "Image upload resolution"